apps/researcher/src/app/[locale]/object/[id]/page.tsx

```diff
@@ -0,0 +1,185 @@
+import {HeritageObject} from '@/lib/heritage-fetcher';
+import {Link} from 'next-intl';
+import Image from 'next/image';
+import {ReactNode} from 'react';
+
+// Revalidate the page
+export const revalidate = 0;
+
+export interface HeaderProps {
+  children: ReactNode;
+}
+
+function H2({children}: HeaderProps) {
+  return <h2 className="font-semibold uppercase text-lg">{children}</h2>;
+}
+
+function H3({children}: HeaderProps) {
+  return <h3 className="font-semibold uppercase text-gray-800">{children}</h3>;
+}
+
+function NotImplemented() {
+  const t = useTranslations('Details');
+
+  return <div>{t('notImplemented')}</div>;
+}
+
+function NoData() {
+  const t = useTranslations('Details');
+
+  return <div>{t('noData')}</div>;
+}
+
+interface InfoBlockProps {
+  object: HeritageObject;
+  objectKey:
+    | 'types'
+    | 'subjects'
+    | 'materials'
+    | 'techniques'
+    | 'creators'
+    | 'inscriptions';
+  singleLine: boolean;
+}
+
+function InfoBlock({object, objectKey, singleLine}: InfoBlockProps) {
+  const t = useTranslations('Details');
+
+  const items = object[objectKey];
+
+  if (!items || !items.length) {
+    return null;
+  }
+
+  const itemValues = items.map(item =>
+    typeof item === 'string' ? item : item.name
+  );
+
+  return (
+    <div>
+      <H3>{t(objectKey)}</H3>
+      {singleLine ? (
+        <>{itemValues.join(', ')}</>
+      ) : (
+        <>
+          {itemValues.map(value => (
+            <div key={value}>{value}</div>
+          ))}
+        </>
+      )}
+    </div>
+  );
+}
+
+interface Props {
+  params: {id: string};
+}
+
+export default async function Details({params}: Props) {
+  const id = decodeURIComponent(params.id);
+  const object = await objectFetcher.getById({id});
+  const t = await getTranslations('Details');
+
+  if (!object) {
+    return <div data-testid="no-object">{t('noObject')}</div>;
+  }
+
+  return (
+    <>
+      <Link href="/">{t('backButton')}</Link>
+      <div className="grid grid-cols-5 mt-5">
+        <div className="col-span-3 flex flex-col space-y-4">
+          <PageHeader>
+            <PageTitle id="about">{object.name}</PageTitle>
+          </PageHeader>
+          <div>
+            <H3>{t('description')}</H3>
+            <div>{object.description}</div>
+          </div>
+          <div>
+            <H3>{t('date')}</H3>
+            <NotImplemented />
+          </div>
+          <div>
+            <H3>{t('owner')}</H3>
+            {object.owner ? (
+              <a href={object.owner.id} target="_blank" rel="noreferrer">
+                {object.owner.name}
+              </a>
+            ) : (
+              <NoData />
+            )}
+          </div>
+          <div>
+            <H3>{t('handle')}</H3>
+            <NotImplemented />
+          </div>
+          <div>
+            <H3>{t('license')}</H3>
+            <NotImplemented />
+          </div>
+          <div className="space-y-4">
+            <H2>{t('moreInfo')}</H2>
```

Comment:
@Doppen For your information:  Based on the design, I have added single-line (joined with a ',') and multi-lined info blocks. The sub-headers are different from the designs.  So I guessed which ones were single- and which were multi-lined. Since we don't have real data, it's probably also hard for you to decide which one to use where. So I suggest we just start with these values and check back when we have more data.